src/components/CurrencyLogo/index.tsx

```diff
@@ -27,13 +27,14 @@ const StyledLogo = styled(Logo)<{ size: string }>`
   border-radius: ${({ size }) => size};
 `
 
-const Wrapper = styled.div<{ size: string; marginRight: number; marginLeft: number; loading?: boolean }>`
+const Wrapper = styled.div<{ size: string; marginRight: number; marginLeft: number; loading?: boolean; top?: number }>`
   position: relative;
   width: ${({ size }) => size};
   height: ${({ size }) => size};
   margin-right: ${({ marginRight }) => marginRight}px;
   margin-left: ${({ marginLeft }) => marginLeft}px;
   border-radius: ${({ size }) => size};
+  top: ${({ top }) => top}px;
 
```

Comment:
I don't know if I'm the only one seeing this but there is white border-like thing around icons. 
![image](https://user-images.githubusercontent.com/26334170/143466794-a3e4ff8b-6b75-4488-bd53-c6d45bc77699.png)
One solution that I tried and it works is to apply `transform: scale(1.05)`. Icons grow a little bit but white border is gone.
![image](https://user-images.githubusercontent.com/26334170/143467056-c63f747e-41c2-4b68-a91f-9467d8333a67.png)